plugins/prometheus/coordinator.go

```diff
@@ -57,13 +65,12 @@ func collectCoordinatorQuorumStats() {
 			"alias":   entry.Alias,
 			"baseURL": entry.BaseURL,
 		}
-		coordinatorQuorumResponseTimes.With(labelsResponseTime).Set(float64(entry.ResponseTimeMs))
+		coordinatorQuorumResponseTimes.With(labelsResponseTime).Observe(entry.ResponseTimeSeconds)
```

Comment:
I am not completely sure but I believe Histograms/Summaries should work differently to gauges where you call observe every time you observed the event and not just when the scrape happens, therefore this should actually be called when a quorum request was done?

https://stackoverflow.com/questions/56297842/what-does-prometheus-summary-observe-method-do

I am also not sure whether the default buckets from `DefBuckets` are within the expected bounds: `DefBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}`

Given that we expect seconds of response latency, the buckets probably need to be adjusted.

Reply:
Done